enginetest/insert_queries.go

```diff
@@ -458,6 +458,48 @@ var InsertScripts = []ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "alter auto_increment value",
+		SetUpScript: []string{
+			`create table auto (
+				pk int auto_increment,
+				c0 int,
+				primary key(pk)
+			);`,
+			"insert into auto values (NULL,10), (NULL,20), (NULL,30)",
+			"alter table auto auto_increment 9;",
+			"insert into auto values (NULL,90)",
+		},
+		Assertions: []ScriptTestAssertion{
+			{
+				Query: "select * from auto",
```

Comment:
Add an order by to all these tests, makes the results in case of failure much more legible